.github/workflows/ci-cd.yml

```diff
@@ -1,6 +1,6 @@
 # This workflow will do a clean install of node dependencies, cache/restore them, build the source code and deploy to staging server by Copying the artifacts using ssh to play.dev.grid.tf
 
-name: CI/CD pipeline 
+name: CI/CD pipeline
```

Comment:
can we please remove any changes in this file?

Reply:
done.